### ISSUE TITLE
Playback from a collection 3

### ISSUE THREAD

**Issue by Ravens62** [project-owner/Peppy#52]
Hi,
With the new version of the player, I can't play from the collection again.
As an audio file it can play, but when I try to play it from the collection it says file not found. There is no Error in the log file, just Debug and Info.

Raspberry Pi4, Waveshare 4.3 DSI, HiFiBerry Amp2
Malevich disk image headless, Czech Language

**Comment by project-owner** [comment]
Here are the use cases I've tested:

- Created a database file on Pi with Kandinsky Edition for the audio files under /home/pi/music directory.
- Installed that database file on Windows desktop and successfully player collection files with Malevich Edition
- Installed the datatbase file on Pi with Malevich Edition and successfully played collection files.

- Created a database file on Pi with Malevich Edition
- Susccessfully played collection files on the same Pi with Malevich Edition

Could you try to create a small database file for example for one directory and see if it works.
Thanks!

**Comment by Ravens62** [comment]
Hi,
it's a collection of music stored on the NAS. I didn't think it had to be listed in the music folder in Peppy. 
I mean, the correct one is "/home/pi/music/Rock"(NAS)
and the wrong one is "/home/pi/music".
I mean, sorry, the playback works.

But if I may ask, I can't search for radio stations.
I mean Country and language, after selecting a fire the player freezes.Genre and title find something, but after marking it the player freezes.

**Comment by project-owner** [comment]
I've tested the Radio Browser functionality. It works fine (except saving the favorites).
Could you check that the volume level is high enough when you select a station?
Another test which you can make - use Radio Browser without display. Use a Web Browser on your desktop or mobile device (http://raspberrypi:8000). This way we can eliminate the display as a root cause.

If it still fails I need all steps to reproduce the issue, right from the startup - what did you do, which items selected in menu.

**Comment by project-owner** [comment]
To fix Radio Browser favorites:
https://github.com/project-owner/PeppyPlayers.doc/wiki/Known-Issues#radio-browser-favorites-doesnt-work

**Comment by Ravens62** [comment]
Hey,
the problem might be in the translation process into the player's chosen language. In Czech it freezes and in English it works. I guess I'll have to mention this in the configuration description...
The first time I found a station, the volume was almost maximum, after switching to another station and searching for another one, the volume has leveled off. 
And the loudness is the same when switching from another player language, in my case from English back to Czech.

And when controlling via web and or display, it freezes as well.

**Comment by project-owner** [comment]
That's probably the issue - missing lables for different languages. Let me check and come back with a fix...

**Comment by project-owner** [comment]
There were those two missing labels for Czech and Dutch. But after adding those lables it worked fine. Could you try to use any other language German or French? Thanks!

**Comment by project-owner** [comment]
Here is the fix for labels:
https://github.com/project-owner/PeppyPlayers.doc/wiki/Known-Issues#radio-browser-for-czech-and-dutch-doesnt-work
Thanks!

**Comment by Ravens62** [comment]
Hi,
I tried using the Malevich edition of Waveshare 7.0 and now it seems to be working. I was using the official 7.0 before but you don't provide that now, so I used headless and that's probably not ideal for me. Still, the max volume surprised me once and that was when switching from collection files to radio.

Raspberry Pi4, Waveshare 4.3 DSI, HiFiBerry Amp2
Malevich Waveshare 7,0" hdmi,LCD, Czech Language

**Comment by project-owner** [comment]
I faced many issues and spent too much time trying to make SPI and DSI displays working with the latest versions of OS, Python and Pygame. I gave up and decided to support only HDMI displays for now. If display makers will provide solutions I'll probably continue making disk images for SPI and DSI displays. Said that, I don't guarantee that SPI and DSI displays will be working with existing disk images. You are lucky if you found a working solution.

I'll try to fix the volume issue during the coming weekend.
Best regards
